### DISCUSSION TITLE
Remove tooltip for column Move

### DISCUSSION THREAD

**Discussion by BenmoussaYounes** [KevinVandy/material-react-table#555] [Q&A] [answered]
I cant find the props to  remove tooltip Move ( for the column Move only )

**Comment by KevinVandy** [comment, marked as answer]
`muiTableHeadCellDragHandleProps.title`

But before removing the tooltip, know that it results in worse accessibility. I don't recommend removing it. Instead, if you need to change the text, you can use the `localization` prop.

https://www.material-react-table.com/docs/guides/localization

**Reply by BenmoussaYounes** [reply]
Works perfectly Thank you
